modular_pariah/modules/contractor/code/datums/midround/antag_datum.dm

```diff
@@ -0,0 +1,57 @@
+// USED FOR THE MIDROUND
+/datum/antagonist/contractor
+	name = "Drifting Contractor"
+	antagpanel_category = "DriftingContractor"
+	preview_outfit = /datum/outfit/contractor_preview
+	job_rank = ROLE_DRIFTING_CONTRACTOR
+	hud_icon = 'modular_pariah/modules/contractor/icons/hud_icon.dmi'
+	antag_hud_name = "contractor"
+	antag_moodlet = /datum/mood_event/focused
+	show_to_ghosts = TRUE
+	suicide_cry = "FOR THE CONTRACTS!!"
+	/// The outfit the contractor is equipped with
+	var/contractor_outfit = /datum/outfit/contractor
+
+/datum/antagonist/contractor/proc/equip_guy()
+	if(!ishuman(owner.current))
+		return
+	var/mob/living/carbon/human/person = owner.current
+	person.equipOutfit(contractor_outfit)
+	return TRUE
```

Comment:
This seems pointless. Does this check not exist on outfit.equip_to()?

Reply:
it does not